docs/source/failure_modes.rst

```diff
@@ -152,33 +152,6 @@ To fix this issue, ensure that all connectors are secured.
 Software Problems
 -----------------
 
-**Shared memory full/Borealis unable to delete shared memory**
```

Comment:
Very satisfying that we can remove this failure mode

Reply:
Agreed, it was a very difficult error to identify.